src/css/custom.css

```diff
@@ -23,3 +23,7 @@
   margin: 0 calc(-1 * var(--ifm-pre-padding));
   padding: 0 var(--ifm-pre-padding);
 }
+
+em {
```

Comment:
This is kinda odd 🤔 
If you're styling some search results, I suggest separating that to `.scss` file next to the `search.tsx`, rather than overwriting styles globally.
❗ **Disclaimer**: maybe I'm missing something, feel free to argue on that one😄